soroban-sdk/src/testutils.rs

```diff
@@ -67,3 +70,21 @@ pub trait Accounts {
     /// Remove an account.
     fn remove(&self, id: &AccountId);
 }
+
+pub fn random_id() -> [u8; 32] {
+    let mut id = [0u8; 32];
+    rand::thread_rng().fill_bytes(&mut id);
+    id
+}
+
+pub fn random_account_id() -> xdr::AccountId {
+    xdr::AccountId(xdr::PublicKey::PublicKeyTypeEd25519(xdr::Uint256(
+        random_id(),
+    )))
+}
+
+impl Env {
+    pub fn random_id_bytes(&self) -> BytesN<32> {
```

Comment:
Could you add Rust docs to these new functions?

Reply:
Added.